catkit/hub/db_status.py

```diff
@@ -0,0 +1,20 @@
+import ase
```

Comment:
Please don't use * imports. This will make reading your code difficult for others and potentially cause import overlaps.